ksql-parser/src/main/java/io/confluent/ksql/parser/AstBuilder.java

```diff
@@ -1118,7 +1072,7 @@ public Node visitParenthesizedExpression(
 
   @Override
   public Node visitCast(final SqlBaseParser.CastContext context) {
-    final boolean isTryCast = context.TRY_CAST() != null;
+    final boolean isTryCast = false;
```

Comment:
Shall we get rid of `isTryCast`? It won't be used anyway.

Reply:
yeah, i think that makes sense, not sure we'll ever want to use this. Removing more code is good!